docs/design-principles.md

```diff
@@ -1,12 +1,12 @@
 We have collated 8 principles which guide AEA framework development:
 
-* Accessibility: easy to use.
-* Modularity: encourages module creation and sharing.
-* Openness: easily extensible with third party libraries.
-* Conciseness: conceptually simple.
-* Value-driven: drives immediate value for some use case.
-* Low entry barriers: leverages existing programming languages and web protocols.
-* Safety: safe for the user (economically speaking).
-* Goal alignment: seamless facilitation of user preferences and goals.
+* **Accessibility**: easy of use.
+* **Modularity**: encourages module creation and sharing and reuse.
+* **Openness**: easily extensible with third party libraries.
+* **Conciseness**: conceptually simple.
+* **Value-driven**: drives immediate value.
+<!--* **Low entry barriers**: leverages existing programming languages and web protocols.-->
+* **Safety**: safe for the user (economically speaking). <!--I suggest changing this to transparency. It is often difficult to show that a system is "safe". There are degrees of safety. What is achievable is in-depth analysis of whatever (e.g. system, method, etc) we propose and transparency about its characteristics (e.g. risk-level associated with a specific method for economic interaction)-->     
```

Comment:
I disagree with changing this to transparency. I don't disagree with your note on that it is difficult to show that a system is "safe", however there are degrees to all of the principles, so I don't see why we should make a difference for this one.

Reply:
It is more about its description; that we aim to make a system safe for the user. 
It is a vague aim that is (probably always) difficult to measure and show whether we've achieved it.

Reply:
Well I think it is quite easy to measure in a lot of cases: e.g. the agent sent you all my tokens even though I didn't get anything from you = very unsafe. Happy for you to provide some alternative description though.